com.playonlinux.PlayOnLinux4.yaml

```diff
@@ -15,6 +15,7 @@ finish-args:
   - --share=network
   - --allow=multiarch
   - --filesystem=host:rw
+  - --env=__GLX_VENDOR_LIBRARY_NAME=mesa
```

Comment:
why this is needed?

Reply:
Without this variable, glxinfo fails with
```
name of display: :0
Error: couldn't find RGB GLX visual or fbconfig
```